.eslintrc.js

```diff
@@ -51,6 +51,9 @@ module.exports = {
       // test files
       files: ['tests/**/*-test.{js,ts}'],
       extends: ['plugin:qunit/recommended'],
+      rules: {
+        'qunit/no-assert-equal': 0,
+      },
```

Comment:
Q: What's the reason for this?

Can we leave it enabled and rebase since this PR? https://github.com/qonto/ember-prismic-dom/pull/19

Reply:
Done! + fixed 2 other `assert.equal`

Reply:
Thanks! 🙏